op-batcher/batcher/driver.go

```diff
@@ -260,9 +260,13 @@ func (l *BatchSubmitter) loadBlocksIntoState(syncStatus eth.SyncStatus, ctx cont
 		return errors.New("start number is >= end number")
 	}
 
+	if end.Number > start.Number+1 {
+		l.Log.Info("loading blocks into state", "start", start.Number+1, "end", end.Number)
+	}
```

Comment:
```suggestion
	if start.Number+1 <= end.Number {
		l.Log.Info("Loading blocks into state (start and end both incl.)", "start", start.Number+1, "end", end.Number)
	}
```

Fixes the `if` to match the below for-loop range (otherwise the `end.Number == start.Number+1` case doesn't have a log), and clarifies the log, since the inclusive-range might be non-obvious to the reader.

Reply:
@protolambda I prefer to not print it when the range length is exactly `1`, as there's already such logs inside `loadBlockIntoState`.